utils/runtime-fuzzer/src/lib.rs

```diff
@@ -74,27 +80,34 @@ fn run_impl(data: &[u8]) -> Result<sp_io::TestExternalities> {
     let test_input_id = get_sha1_string(data);
     log::trace!("Generating GearCalls from corpus - {}", test_input_id);
 
+    let fuzzing_config = default_fuzzing_config();
+
     let sender = runtime::account(runtime::alice());
     let sender_prog_id = ProgramId::from(*<AccountId as AsRef<[u8; 32]>>::as_ref(&sender));
 
+    let mut unstructured = Unstructured::new(data);
+
+    let initial_sender_balance =
+        unstructured.int_in_range(fuzzing_config.initial_sender_balance.clone())?;
+
     let generators = default_generator_set(test_input_id);
-    let gear_calls = GearCalls::new(data, generators, vec![sender_prog_id])?;
+    let gear_calls = GearCalls::new(unstructured, generators, vec![sender_prog_id])?;
 
     let mut test_ext = new_test_ext();
     test_ext.execute_with(|| -> Result<()> {
-        // Increase maximum balance of the `sender`.
+        // Set balance of the `sender`.
         {
-            increase_to_max_balance(sender.clone())
+            set_account_balance(sender.clone(), initial_sender_balance)
```

Comment:
balance will be burned for: tx commission (not sure how do you call txs, so this one is optional), gas purchase and value sending, so seems you need to somehow handle it

Reply:
_fuzzer related comments_
resolving, a new PR for fuzzer changes will be opened.